src/containers/admin-dashBoard/ManagerRoles.tsx

```diff
@@ -211,13 +222,17 @@ const AdminSission = () => {
                       </tr>
                     </thead>
                     <tbody>
-                      <tr>
-                        <td className="border-b border-gray-200 bg-white dark:bg-dark-bg">
-                          <div className="flex items-left">
-                            <div className="md:grid md:grid-cols-2">
-                              <p></p>
+                      {rolemanagement.map((role: any, index: any) => {
```

Comment:
Is it a role or a user. Please change the naming to ensure that everyone is aware of the object that is being destructured.

We can not expect to have a role that has a name attribute